### DISCUSSION TITLE
Battle #1 #4 Ups and Downs why such a low score if output it's identical ?

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#114] [General]
https://imgur.com/a/191rYKC
can someone explain to me, why my solution got just 70%?
it's pixel perfect align with the target,
0 difference when I slide and compare,

unfair 👎

**Comment by chinchang** [comment]
Hi! We are checking this. I am assuming the code shown in the shared image is complete code?

**Reply by chinchang** [reply]
Hi @rafehimself You are probably playing on a non-chromium browsers because this is how your code renders on chrome:
![image](https://user-images.githubusercontent.com/379918/135727546-59160842-a4ec-42f3-a0b7-78bf8ac20e97.png)

**Reply by rafehimself** [reply]
not at all I use Chrome,  94.0.4606.71 version 64 bit,

```
<div class="a"></div>
<div class="b"></div>
<div class="c"></div>
<style>
body {
   margin: 50px 0;
   background: #62306D;
   display: flex;
   justify-content: center;
   align-items: end;
 }
div {
    width: 100px;
    height: 100px;
    background: #F7EC7D;
    border-radius: 50% 50% 0 0;
}
.b {
align-self: start;
}
.a, .c {
transform: scaleY(-1);
}
</style>
```
I think the problem might be that it works only on those box-window values, if I stretch that default box 400-300 the output will change, cuz they are not absolute, I experienced that with gird as well, in battle#1 #2 when I used grid I got less point than when I used absolute values, regardless of output being the same in that 400x300 box

I don't see any other problem with that code

**Reply by chinchang** [reply]
Ah I see. So the issue here gets fixed when you use `flex-end` instead of `end`. `end` is an upcoming cross layout value, so might be the we need to upgrade our backend chromium. We'll try and update here.

**Reply by chinchang** [reply]
this is fixed now!

**Comment by rafehimself** [comment]
![image](https://user-images.githubusercontent.com/58576132/135799286-ea2903bc-d159-4c49-a3f2-d17f66152fea.png)
`<p o><p><p i><style>body{background:#0B2429;display:grid;place-items:center;}p{width:150;height:150;background:#998235;border-radius:100px 0 100px;z-index:2}p[i]{background:#F3AC3C;position:fixed;right:75}p[o]{background:#1A4341;position:fixed;left:75`
another example, its clearly broken somehow

sorry for the code, but you made those silly rules 😝

**Reply by rafehimself** [reply]
![image](https://user-images.githubusercontent.com/58576132/135814105-a53c6c80-5455-4b1f-8de4-490589da2dfa.png)
problem with grid as well

**Reply by chinchang** [reply]
fixed now :)

**Comment by chinchang** [comment]
We updated chrome to latest version. These issues should be fixed now.

**Comment by rafehimself** [comment]
Yeah it works now 👍 great, I really enjoy this whole project. 😃  Thank you

**Comment by optyler** [comment]
Hi guys, still an issue I guess, getting 99.8% ... did i miss something or a bug still exists ?
```
<p><p u><p><style>*{background:var(--,#62306D)}body{padding: 127px 0 0 42px}* p {display: inline-block;border-radius: 0 0 50% 50%;--:#F7EC7D;width:100;height:100}* p[u]{transform: scaleY(-1)translateY(100%)
```

**Reply by chinchang** [reply]
minor deviations like these happen mostly due to minor mismatches in few pixels. Make sure there are no pixels bleading out of clip-paths or pixels at the intersection of two curves.

**Reply by chinchang** [reply]
BTW, I checked your code and there is visible deviation in the output and target. Make sure you are on a chromium-based browser :)

**Reply by optyler** [reply]
so... i miss something ^^' thx for you comment :)

**Reply by rafehimself** [reply]
try to fix your padding in body, there is 1px difference
